amundsen_application/static/js/components/common/ResourceListItem/UserListItem/index.tsx

```diff
@@ -0,0 +1,50 @@
+import * as React from 'react';
+import Avatar from 'react-avatar';
+import { Link } from 'react-router-dom';
+
+import { LoggingParams, UserResource} from '../types';
+
+interface UserListItemProps {
+  user: UserResource;
+  logging: LoggingParams;
+}
+
+class UserListItem extends React.Component<UserListItemProps, {}> {
+  constructor(props) {
+    super(props);
+  }
+
+  getLink = () => {
+    const { user, logging } = this.props;
+    return `/user/${user.id}/?index=${logging.index}&source=${logging.source}`;
+  };
+
+  render() {
+    const { user } = this.props;
+    return (
+      <li className="list-group-item">
+        <Link className="resource-list-item user-list-item" to={ this.getLink() }>
+          <Avatar name={ user.name } size={ 24 } round={ true } />
+          <div className="content">
+            <div className="col-xs-12 col-sm-6">
+              <div className="main-title">{ user.name }</div>
```

Comment:
We're missing support for the `Flag` component next to the name if the user is not active.